### ISSUE TITLE
Publish Maven mojo docs 

### ISSUE THREAD

**Issue by ppalaga** [quarkusio/quarkusio.github.io#265]
We have several maven plugins in the source tree and their documentation pages (as generated by maven site plugin) are not published anywhere. Users (including me) expect to find all available params, their defaults, etc there. It would be quite tedious to re-document all that stuff in asciidoc.

**Comment by maxandersen** [comment]
good point - now if only we could get those pages generated as .adoc rather than html and we could then easily post them like we sync over guides to the website ;)

**Comment by gsmet** [comment]
I think the best would be to generate a Maven site and push it to the website as part of the release.

@maxandersen if you can find a good URL and where we could put the links to it, I can have a look for an upcoming release.

**Comment by maxandersen** [comment]
agreed @gsmet - was looking at in the current structure it would fit best under something like "/guides/<maven plugin>" but  still rubs me the wrong way.

Especially since it doesn't just stop with maven plugins, also gradle tasks, core and extension api docs would be nice to have a "place" for - and on top those would be version bound; but I reckon to start we could just start with a `/docs/latest/[api|maven|...]` 

çonclusion is though that for now these version specific docs don't really fit in somewhere in the site.

**Comment by maxandersen** [comment]
@emmanuelbernard was any thoughts done on where to put api/version dependent docs during initial site stucture ?
